server/graphql/v1/mutations/orders.js

```diff
@@ -156,6 +156,22 @@ export async function createOrder(order, loaders, remoteUser) {
         );
       }
 
+      // if there are more than 5 transactions in the last hour, blocks new orders
+      const end = new Date();
+      const start = new Date();
+      start.setHours(start.getHours() - 1);
+      const where = {
+        FromCollectiveId: order.fromCollective.id,
+        createdAt: { [Op.between]: [start, end] },
+        type: 'CREDIT',
+      };
+      const lastHourTransactionsCount = await models.Transaction.count({ where });
+      if ( lastHourTransactionsCount >= 5 ) {
```

Comment:
>=3

No reason to have more than 3 transactions from and to the same collective.

Reply:
ah I didn't realize it was not to the same collective.
Maybe add `CollectiveId` in the `where`?